src/AutorestSwift/Utils/ViewModelUtil.swift

```diff
@@ -0,0 +1,39 @@
+// --------------------------------------------------------------------------
+//
+// Copyright (c) Microsoft Corporation. All rights reserved.
+//
+// The MIT License (MIT)
+//
+// Permission is hereby granted, free of charge, to any person obtaining a copy
+// of this software and associated documentation files (the ""Software""), to
+// deal in the Software without restriction, including without limitation the
+// rights to use, copy, modify, merge, publish, distribute, sublicense, and/or
+// sell copies of the Software, and to permit persons to whom the Software is
+// furnished to do so, subject to the following conditions:
+//
+// The above copyright notice and this permission notice shall be included in
+// all copies or substantial portions of the Software.
+//
+// THE SOFTWARE IS PROVIDED *AS IS*, WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+// IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+// FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+// AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+// LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING
+// FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS
+// IN THE SOFTWARE.
+//
+// --------------------------------------------------------------------------
+
+import Foundation
+
+import Stencil
+
+public func getStrategy(_ objectType: String) -> String {
+    if objectType == "String" {
+        return ResponseBodyType.stringBody.rawValue
+    } else if objectType.starts(with: "Int") {
+        return ResponseBodyType.intBody.rawValue
+    } else {
+        return ResponseBodyType.jsonBody.rawValue
+    }
+}
```

Comment:
This seems like it should be a function on the enum type. 

```swift
enum ResponseBodyType {
  func strategy(for input: String) -> ResponseBodyType  {
    switch input {
      case "String":
        return .stringBody
      // etc
    }
  }

}
```

Reply:
Thanks. Changed per suggestion.